src/score_hv/hv_registry.py

```diff
@@ -8,14 +8,15 @@
 from collections import namedtuple
 from score_hv.harvesters.innov_netcdf import InnovStatsCfg, InnovStatsHv
 from score_hv.harvesters.obs_log import ObsInfoCfg, ObsInfoHv
-
+from score_hv.harvesters.inc_logs import AtmIncCfg
```

Comment:
This change probably shouldn't be in this PR if you are treating it as just fixing the tests, along with the other inc_logs references

Reply:
Agreed. Thanks for pointing this out to me.